client/src/views/Explore.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
+import {View, Text, Button, StyleSheet} from 'react-native';
+import {goToFirstScreenInStack, navigateTo} from '../helpers/NavigateHelper';
+
+class Explore extends Component {
+  render() {
+    const {navigation} = this.props;
+    return (
+      <View style={styles.container}>
+        <Text style={styles.text}>This is Explore.</Text>
+        <View style={styles.buttonWrapper1}>
+          <Button
+            title="Go to Review"
+            onPress={() => navigateTo(navigation, 'Review')}
+          />
+          <Button
+            title="Go back Home"
+            onPress={() => goToFirstScreenInStack(navigation)}
+          />
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  text: {
+    fontSize: 20,
+    paddingBottom: 15,
+  },
+  buttonWrapper1: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
```

Comment:
we should add template Redux here (mapPropsfromstate, mapPropsfromreducer...)
and for all our three components.
It may help for later development.

Reply:
true, but we will use them when we need them, for now I don't think we need to have those two methods and connect it to redux state

Reply:
if the template have it, we can remove it if we don't need it. that way will enhance the speed of developing

Reply:
right, but I imagine that there will be more components inside those screens, they might be nested and I think whichever component needs it, will import it

Reply:
I used to pass props through Stack... as params. it was super complicated. we need redux anyway.

Reply:
ok